src/main/java/com/github/ygimenez/method/Pages.java

```diff
@@ -724,7 +725,7 @@ public static void categorize(@Nonnull Message msg, @Nonnull Map<Emoji, Page> ca
 		if (useBtns) {
 			List<ActionRow> rows = new ArrayList<>();
 
-			List<Component> row = new ArrayList<>();
+			List<ActionComponent> row = new ArrayList<>();
```

Comment:
Would be better to refrain from using subclass types since the library might support Components other than Button futurely.